tortoise/models.py

```diff
@@ -297,5 +309,37 @@ def get(cls, *args, **kwargs) -> QuerySet:
             db=db,
         ).fetch_for_list(instance_list, *args)
 
+    @classmethod
+    def check(cls):
+        cls._check_unique_together()
+
+    @classmethod
+    def _check_unique_together(cls):
+        """Check the value of "unique_together" option."""
+        if cls._meta.unique_together is None:
+            return
+
+        if not isinstance(cls._meta.unique_together, (tuple, list)):
+            raise ConfigurationError(
+                f"'{cls.__name__}.unique_together' must be a list or tuple.")
```

Comment:
We still target Py3.5 compatibility, please change this to use `.format()` instead.